sound/soc/sof/ipc4-pcm.c

```diff
@@ -276,6 +283,36 @@ static int sof_ipc4_chain_dma_trigger(struct snd_sof_dev *sdev,
 	return sof_ipc_tx_message_no_reply(sdev->ipc, &msg, 0);
 }
 
+static void sof_ipc4_sort_pipeline(struct ipc4_pipeline_set_state_data *trigger_list,
```

Comment:
@RanderWang instead of doing this can we insert the pipelines in the trigger_list in the descending order of priorities.

Reply:
sure, good idea!  I first want to reserve original logic if priority is not set. We can achieve it with your suggestion. Thanks!